src/key_value_store/server/kvstore_server_runner.cpp

```diff
@@ -0,0 +1,26 @@
+#include "kvstore_server.h"
+
+void RunServer() {
+  std::string server_address("0.0.0.0:50001");
+  KeyValueStoreImpl service;
+
+  ServerBuilder builder;
+  // Listen on the given address without any authentication mechanism.
+  builder.AddListeningPort(server_address, grpc::InsecureServerCredentials());
+  // Register "service" as the instance through which we'll communicate with
+  // clients. In this case it corresponds to an *synchronous* service.
+  builder.RegisterService(&service);
+  // Finally assemble the server.
+  std::unique_ptr<Server> server(builder.BuildAndStart());
+  std::cout << "Server listening on " << server_address << std::endl;
+
+  // Wait for the server to shutdown. Note that some other thread must be
+  // responsible for shutting down the server for this call to ever return.
+  server->Wait();
+}
+
+int main(int argc, char **argv) {
+  RunServer();
+
+  return 0;
+}
```

Comment:
Merge this into the same executable build target as the other, empty main in the kvstore folder

Reply:
Done!